ui/lobby/src/setupCtrl.ts

```diff
@@ -149,9 +148,45 @@ export default class SetupController {
       aiLevel: this.aiLevel(),
     });
 
+  private savePropsToStoreExceptRating = () => {
+    if (this.gameType && this.store[this.gameType]) {
+      const ratingMin = this.store[this.gameType]().ratingMin;
+      const ratingMax = this.store[this.gameType]().ratingMax;
+      this.store[this.gameType]({
+        variant: this.variant(),
+        fen: this.fen(),
+        timeMode: this.timeMode(),
+        time: this.time(),
+        increment: this.increment(),
+        days: this.days(),
+        gameMode: this.gameMode(),
+        ratingMin: ratingMin,
+        ratingMax: ratingMax,
+        aiLevel: this.aiLevel(),
+      });
+    }
+  };
```

Comment:
surely that can be refactored to remove duplicated code with `savePropsToStore`

Reply:
How can I reuse savePropsToStore so that I do not store updated value